### PR TITLE
doc(pu): add ppo benchmark results

### PR DESCRIPTION
**Pull request by puyuan1996**
add ppo benchmark results

### CODE REVIEW

File: source/hands_on/ppo.rst [hunk 1]
```diff
@@ -90,9 +90,111 @@ The policy gradient and value update of PPO is implemented as follows:
...
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_ha <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Halfcheetah          |                 |                                                     |DI-engine/tree/main/dizoo/|   Tianshou(7194)     |
+|                     |  3500           |.. image:: images/benchmark/halfcheetah_onppo.png    |mujoco/config/serial/     |     Sb3(1976)        |
+|(Halfcheetah-v3)     |                 |                                                     |halfcheetah/halfcheetah   |                      |
+|                     |                 |                                                     |_onppo_config.py>`_       |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
```
Comment (Will-Nie) [inline source/hands_on/ppo.rst:196]: may also add spinning up for comparison for mujoco

File: source/hands_on/ppo.rst [hunk 2]
```diff
@@ -90,9 +90,111 @@ The policy gradient and value update of PPO is implemented as follows:
...
+|                     |                 |                                                     |.py>`_                    |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_s <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|SpaceInvaders        |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  400            |.. image:: images/benchmark/spaceinvaders_onppo.png  |atari/config/serial/      |                      |
+|(SpaceInvadersNoFrame|                 |                                                     |spaceinvaders/spacein     |                      |
+|skip-v4)             |                 |                                                     |vaders_onppo_config.py>`_ |                      |
```
Comment (Will-Nie) [inline source/hands_on/ppo.rst:174]: may also add sb3 benchmark for atari

File: source/hands_on/ppo.rst [hunk 3]
```diff
@@ -90,9 +90,111 @@ The policy gradient and value update of PPO is implemented as follows:
...
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_ha <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Halfcheetah          |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  2000           |.. image:: images/benchmark/halfcheetah_offppo.png   |mujoco/config/serial/     |                      |
+|(Halfcheetah-v3)    |                 |                                                     |halfcheetah/halfcheetah   |                      |
+|                     |                 |                                                     |_offppo_config.py>`_      |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
```
Comment (Will-Nie) [inline source/hands_on/ppo.rst:146]: This table does not display

File: source/hands_on/ppo.rst [hunk 4]
```diff
@@ -90,9 +90,111 @@ The policy gradient and value update of PPO is implemented as follows:
...
+|                     |                 |                                                     |`config_link_ha <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Halfcheetah          |                 |                                                     |DI-engine/tree/main/dizoo/|   Tianshou(7194)     |
+|                     |  3500           |.. image:: images/benchmark/halfcheetah_onppo.png    |mujoco/config/serial/     |     Sb3(1976)        |
+|(Halfcheetah-v3)     |                 |                                                     |halfcheetah/halfcheetah   |                      |
+|                     |                 |                                                     |_onppo_config.py>`_       |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
```
Comment (Will-Nie) [inline source/hands_on/ppo.rst:197]: Is 'What Matters In On-Policy Reinforcement Learning?
A Large-Scale Empirical Study' also referred?

Comment (Will-Nie) [inline source/hands_on/ppo.rst:197]: Total commets:
In line 26, L^{C L I P}(\theta)=\hat{E}_{t}\left[\min \left(r_{t}(\theta) \hat{A}_{t}, \operatorname{clip}\left(r_{t}(\theta), 1-\epsilon, 1+\epsilon\right) \hat{A}_{t}\right)\right] will not report error in LaTex

File: source/hands_on/ppo.rst (hunk 2 above)
Comment (puyuan1996) [reply source/hands_on/ppo.rst:174]: I can't find sb3 atari onppo results.

File: source/hands_on/ppo.rst [hunk 5]
```diff
@@ -148,23 +149,23 @@ The interface of ``ppo_policy_error`` and ``ppo_value_error`` is defined as foll
...
+     - | In the paper `Mastering Complex Control in MOBA Games with Deep Reinforcement Learning <https://arxiv.org/abs/1912.09729>`_, the authors claim that when :math:`\hat{A}_t < 0`, a too large :math:`r_t(\theta)` should also be clipped, which introduces dual clip: .. math:: \max \left(\min \left(r_{t}(\theta) \hat{A}_{t}, {clip}\left(r_{t}(\theta), 1-\epsilon, 1+\epsilon\right) \hat{A}_{t}\right), c \hat{A}_{t}\right)
    * - | Recompute Advantage
        |
-     - In on-policy PPO, each time we collect a batch data, to improve data efficiency, we will train many epochs, before the beginning of each training epoch, we recompute the advantage of historical transitions, to keep the estimation of advantage close to current policy.
+     - | In on-policy PPO, each time we collect a batch data, to improve data efficiency, we will train many epochs, before the beginning of each training epoch, we recompute the advantage of historical transitions, to keep the estimation of advantage close to current policy.
```
Comment (chenyingvivian) [inline source/hands_on/ppo.rst]: 将to improve data efficiency放到we will train many epochs...这句话后面更好些，即“we will train many epochs before the beginning of each training epoch to improve data efficiency, and we will recompute the advantage of historical transitions to keep the estimation of advantage close to current policy."

File: source/hands_on/ppo.rst [hunk 6]
```diff
@@ -148,23 +149,23 @@ The interface of ``ppo_policy_error`` and ``ppo_value_error`` is defined as foll
...
    * - | Value/Advantage Normalization
        |
-     - We standardize the targets of the value/advantage function by using running estimates of the average and standard deviation of the value/advantage targets to.
+     - | We standardize the targets of the value/advantage function by using running estimates of the average and standard deviation of the value/advantage targets to.
```
Comment (chenyingvivian) [inline source/hands_on/ppo.rst]: to what? 感觉这句话没说完又或者可以把to删除

File: source/hands_on/ppo.rst [hunk 7]
```diff
@@ -148,23 +149,23 @@ The interface of ``ppo_policy_error`` and ``ppo_value_error`` is defined as foll
...
        For more implementation details about recompute advantage and normalization, users can refer to this `discussion <https://github.com/opendilab/DI-engine/discussions/172#discussioncomment-1901038>`__.
    * - | `Value Clipping <https://github.com/opendilab/DI-engine/blob/e6cc06043b479b164b41189ac99c9315c0c938de/ding/rl_utils/ppo.py#L202>`_
        |
-     - Value is clipped around the previous value estimates and use the clip_ratio same as that used to clip probability ratios in the PPO policy loss function.
+     - | Value is clipped around the previous value estimates and use the clip_ratio same as that used to clip probability ratios in the PPO policy loss function.
```
Comment (chenyingvivian) [inline source/hands_on/ppo.rst]: ...and uses the clip_ratio （或者应该是clip-ratio?), which is same as that used to...

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**